clients/spark/core/src/main/scala/io/treeverse/gc/UncommittedGarbageCollector.scala

```diff
@@ -199,11 +195,6 @@ object UncommittedGarbageCollector {
           .toDF()
         removed.collect()
       }
-    } catch {
-      case e: Throwable =>
-        success = false
```

Comment:
Nice - just initialize `success` with false and set it to true at the end of the 'try' scope.